app/scripts/controllers/detect-tokens.js

```diff
@@ -114,6 +115,7 @@ export default class DetectTokensController {
       : tokenList;
 
     const tokensToDetect = [];
+    this.ethersProvider = new Web3Provider(this._network._provider);
```

Comment:
Oh, I don't know if this file needs a provider any more?

Reply:
looks like you're right! Will remove

Reply:
done [here](https://github.com/MetaMask/metamask-extension/pull/15461/commits/cbe7652b41a0835def382ee508550348e23679ce)